function1.js

```diff
@@ -0,0 +1,32 @@
+
+
+function angle (aNum) {
+       if ( aNum === 90) {
+        return "It's a right Angle";
+    } else if (aNum <= 90 && aNum > 0) {
+        return"It's an acute Angle";
+    } else if (aNum == 180) {
+        return"It's a straight Angle";
+    } else if (aNum < 180 && aNum > 90) {
+        return"It's a obtuse Angle";
+    } else if (aNum >180 && aNum < 360) {
+        return"It's a reflex Angle";
+    } else if (aNum == 360) {
+        return "It's a Circle ";
+    } else {
+        return"This is not a valid number";
+    }
+}
+
+ 
+console.log(angle());
+console.log(angle(60));
```

Comment:
If you make the change above, you'll have to change these to:
```js
console.log("It's a "+ angle());
console.log("It's a "+ angle(60));
```
or
```js
console.log("It's a %s", angle());
console.log("It's a %s", angle(60));
```
(see [Using String Substitutions](https://developer.mozilla.org/en-US/docs/Web/API/console#Using_string_substitutions) with console.log)